### ISSUE TITLE
prisma generate does not process multiple env variables of the prisma.yml endpoint

### ISSUE THREAD

**Issue by Inkdpixels** [prisma/prisma#3192]
**Describe the bug**
As of version `1.17.1` the command `prisma generate` does fail if the `prisma.yml` endpoint holds a template literal such as `${env:CORE_REVISION}`, an example config can be found below.

```yml
endpoint: http://${env:CORE_PRISMA_HOST}:4466/service/v${env:CORE_REVISION}
datamodel:
  - model/Category.graphql
generate:
  - generator: typescript-client
    output: ./client'
```

**To Reproduce**
1. Install prisma
2. Setup the config as posted above
3. Execute `prisma generate`

```
Generating schema... 32ms
SyntaxError: '}' expected. (1478:152)
  1476 | */
  1477 | 
> 1478 | export const Prisma = makePrismaClientClass<ClientConstructor<Prisma>>({typeDefs, endpoint: `http://${process.env['CORE_PRISMA_HOST']}:4466/core/v${env:CORE_REVISION}`})
       |                                                                                                                                                        ^
  1479 | export const prisma = new Prisma()
```

**Expected behavior**
The env variable / template literal should be processed and not cause any errors.

**Versions (please complete the following information):**
 - OS: alpine linux
 - `prisma` CLI: `prisma/1.17.1 node-v10.4.0`

**Comment by marktani** [comment]
It works for one env var, but the second env var seem to confuse the parsing.

**Comment by divyenduz** [comment]
This fix is available in the stable version - 1.18.1 - `npm install -g prisma` 🎉
